frontends/qt/resources/Makefile

```diff
@@ -0,0 +1,9 @@
+SIZES=16x16 32x32 48x48 64x64 128x128
+
+TARGETS=$(addsuffix /apps/bitbox.png, $(addprefix linux/usr/share/icons/hicolor/, ${SIZES}))
```

Comment:
Could you swap `addsuffix` and `addprefix`? It just really hurts reading "addsuffix" first, then "addprefix" and swapping them around in my head. :)

Reply:
Done, PTAL

Reply:
:+1: thanks!